lib/utils.bash

```diff
@@ -19,21 +16,12 @@ asdf_version() {
   fi
 }
 
-asdf_dir() {
-  if [ -z "$ASDF_DIR" ]; then
-    local current_script_path=${BASH_SOURCE[0]}
-    export ASDF_DIR
-    ASDF_DIR=$(
-      cd "$(dirname "$(dirname "$current_script_path")")" || exit
-      printf '%s\n' "$PWD"
-    )
-  fi
-
-  printf "%s\n" "$ASDF_DIR"
+asdf_default_tool_versions_filename() {
```

Comment:
nitpick: The default tool versions file name is `.tool-versions`. If the user has customized it by setting `ASDF_DEFAULT_TOOL_VERSIONS_FILENAME` then this function won't return the default filename. I think `asdf_tool_versions_filename` would be a better function name.